### PR TITLE
Split form study details

### LINKED ISSUE #327: UPDATE, STRUCTURE - Update study details pages for new split form
See https://github.com/lookit/lookit-api/pull/1244

### PR DESCRIPTION
**Pull request by okaycj**
# Summary

Fixes #327

This PR covers a few issues:
- Adds a sphinx development server to the make file and requirements.
- Update language around the "Create Study" button.
- Update screenshots of study details page.
- Update screenshots/language on updating EFP's commit SHA
- A few other small fixes here and there

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -17,4 +17,7 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
-	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
\ No newline at end of file
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+livehtml:
+	sphinx-autobuild --port=9999 "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```
Comment (becky-gilbert) [inline docs/Makefile:23]: Yay, thanks!

**Review by becky-gilbert — APPROVED** [review]
Great!

File: docs/source/_static/img/study_detail.png
Comment (mekline) [inline docs/source/_static/img/study_detail.png:1]: For examples like this where the new screenshot shows less content than the original, have you checked that the missing bits aren't referred to in the text? That is, making sure that the text doesn't talk about e.g. changing the study state when the screenshot now cuts off at the study link?  

I think it's a good call to use these simpler screenshots with less content, just need to check that we didn't misplace any signposts as we do it!

File: docs/source/_static/img/study_edit.png
Comment (mekline) [inline docs/source/_static/img/study_edit.png:1]: (This was the other image I wondered about the text context for - I like the shorter screenshot, but check that the surrounding text doesn't assume that e.g. a pic of the exit url field is visible. If it does we probably just need to do some text massaging, let me know)

File: docs/source/researchers-manage-studies.rst [hunk 2]
```diff
@@ -179,4 +179,8 @@ For external studies, the "Edit Study Design" page allows you to edit the follow
 
 .. _`Setup for custom frame development`: frame-dev-setup.html
 
-.. _`Setting study fields`: researchers-set-study-fields.html
\ No newline at end of file
+.. _`Setting study fields`: researchers-set-study-fields.html
+
+.. _`study list page`: https://childrenhelpingscience.com/exp/studies/
```
Comment (becky-gilbert) [inline docs/source/researchers-manage-studies.rst:184]: Do know if these links names/references are case-sensitive? In "study list page", the "s" is lower case here and upper case where it's referenced in the text above. I had a quick look in the sphinx rst docs here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks but couldn't find a quick answer.

**Review by becky-gilbert — APPROVED** [review]
Great work!!

**Comment by becky-gilbert** [comment]
@okaycj do you know if you updated the python version in your docs virtual environment? I tried updating my environment with the updated requirements doc in this PR, and I get some errors about ignoring certain packages/versions that are not compatible with python 3.8.

Screenshot 2024-12-13 at 2 49 20 PM
<br><br>

I tried just running `make livehtml` anyway but that didn't work. It did work after I installed sphinx autobuild directly: `pip install sphinx-autobuild`.

Is this just me doing something wrong? Or if we are updating python, then we should change the [readme installation instructions](https://github.com/lookit/lookit-docs?tab=readme-ov-file#installation), which say to use 3.8.

**Pull request merged**